llvm/include/llvm/SandboxIR/Tracker.h

```diff
@@ -213,6 +213,25 @@ class CallBrInstSetIndirectDest : public IRChangeBase {
 #endif
 };
 
+class SetVolatile : public IRChangeBase {
+  Instruction *Inst;
```

Comment:
I think we don't need an extra `Instruction *` variable. We already have the `InstUnion` to hold the Load or Store we are about to modify.

Reply:
That makes sense, thanks for the suggestion!